### PR TITLE
Only show wifi skip warning if not connected

### PR DESCRIPTION
**Pull request by angusjfw**
| Status  | Ticket/Issue |
| :---: | :--: |
| Ready/Hold | [Ticket](https://pi-top.atlassian.net/browse/<ticket-number>) |

#### Main changes
-

#### Screenshots (feature, test output, profiling, dev tools etc)

[insert screenshots here]

#### Other notes (e.g. implementation quirks, edge cases, questions / issues)
-

#### Manual testing tips
-

#### Tag anyone who definitely needs to review or help
-

### CODE REVIEW

File: frontend/src/pages/wifiPage/WifiPage.tsx [hunk 1]
```diff
@@ -82,7 +82,11 @@ export default ({
           onClick: onNextClick,
           disabled: !isConnected,
         }}
-        skipButton={{ onClick: () => setIsSkipWarningDialogActive(true) }}
+        skipButton={{ onClick: () => {
+          if (!isConnected) {
+            setIsSkipWarningDialogActive(true)
+          }
```
[the remark this answers is not in the document]
Comment (angusjfw) [reply frontend/src/pages/wifiPage/WifiPage.tsx:88]: oh yeah sorry i think this is what I intended

[the remark this answers is not in the document]
Comment (angusjfw) [reply frontend/src/pages/wifiPage/WifiPage.tsx:88]:
```suggestion
          if (!isConnected) {
            setIsSkipWarningDialogActive(true)
          } else {
            onSkipClick();
          }
```

**Review by jcapona — APPROVED** [review]
